definitions.js

```diff
@@ -71,3 +72,9 @@ export const PURCHASE_CATEGORY_ENTERTAINMENT_BAR_NIGHTCLUB = 'Bar or nightclub';
 export const PURCHASE_CATEGORY_ENTERTAINMENT_GAMBLING = 'Gambling';
 export const PURCHASE_CATEGORY_ENTERTAINMENT_CRUISE_LINES = 'Cruise';
 export const PURCHASE_CATEGORY_ENTERTAINMENT_LIQUOR_STORE = 'Liquor store';
+
+// Hotel  stay
+export const HOTEL_CLASS_ZERO_TO_TWO_STARS = 'Zero to two stars hotel';
+export const HOTEL_CLASS_THREE_STARS = 'Three stars hotel';
+export const HOTEL_CLASS_FOUR_STARS = 'Four stars hotel';
+export const HOTEL_CLASS_FIVE_STARS = 'Five stars hotel';
```

Comment:
Can we make these a bit more like ids?
In the api fx you would have to write 
```javascript
"activity": {
	"activityType": "ACTIVITY_TYPE_LODGING",
	"hotelClass": "Zero to two stars hotel",
	"durationHours": 24
}
```
And if you miscapitazlize one letter, it won't work

Maybe just write 
```javascript
HOTEL_CLASS_ZERO_TO_TWO_STARS = 'HOTEL_CLASS_ZERO_TO_TWO_STARS'
```